Resources/views/Configuration/config.js.twig

```diff
@@ -44,6 +44,8 @@
                 .icon('<span class="glyphicon glyphicon-{{ entity.icon }}"></span>');
 
             {{ entity.name }}.dashboardView()
+                .title('Recent {{ entity.name }}')
```

Comment:
The reason I didn't is because with 0.7.0 LemonRestBundle will default names to plural.  I did this after some discussion with Symfony Rest group about best practices.  Consequently, they'll be plural by default after the 0.7.0 upgrade.  I can go ahead and change this if you like, or we can just wait until 0.7.0 gets out. Let me know.